### DISCUSSION TITLE
scroll.scroll_to(0, 0)   is don't work.

### DISCUSSION THREAD

**Discussion by ivan-hz** [fltk-rs/fltk-rs#1345] [Q&A]
I found    scroll.scroll_to(0, 0)   is don't work. Does anyone know why????? is there something wrong with me????

but if i  put the button inside and click it is ok.
    

let dx = scroll.xposition();
    let dy = scroll.yposition();
    for i in 0..20 {
        let mut table = input::Input::default();
        table.set_size(50, 30);
        table.set_pos(dx, dy + (i * 30));
        table.set_value(&i.to_string());
        scroll.add(&table);
        
    }
    let dx = scroll.xposition();
    let dy = scroll.yposition();
    for i in 0..20 {
        let mut table = input::Input::default();
        table.set_size(50, 30);
        table.set_pos(dx, dy + (i * 30));
        table.set_value(&i.to_string());
        scroll.add(&table);
        
    }
  
    **scroll.scroll_to(0, 0);   //don't work**

    ui.ck.set_callback(move |_| {
        println!("{:?}", scroll.scrollbar().value());
        **scroll.scroll_to(0, 0); // It work**

    });

**Comment by MoAlyousef** [comment]
When creating a Scroll widget, it's already at position 0, 0. If you scroll down then scroll_to(0, 0) will work. 
```rust
use fltk::{prelude::*, *};

fn main() {
    let app = app::App::default().with_scheme(app::Scheme::Gtk);
    let mut win = window::Window::default().with_size(400, 300);
    let mut scroll = group::Scroll::default().with_size(400, 300);
    scroll.end();
    win.end();
    win.show();

    for i in 0..20 {
        let b = button::Button::new(100, i * 30, 80, 30, None).with_label(&i.to_string());
        scroll.add(&b);
    }

    scroll.scroll_to(0, 30 * 5); // scroll to the fifth button

    app.run().unwrap();
}
```
